rmw_fastrtps_dynamic_cpp/src/TypeSupport_impl.hpp

```diff
@@ -812,6 +916,24 @@ bool TypeSupport<MembersType>::deserializeROSmessage(
 template<typename MembersType>
 size_t TypeSupport<MembersType>::calculateMaxSerializedSize(
   const MembersType * members, size_t current_alignment)
+{
+  bool is_key_unbounded{false};  // unused
+  return calculateMaxSerializedSize(members, current_alignment, false, is_key_unbounded);
```

Comment:
why do we not just put the `false` in the argument?

```suggestion
  return calculateMaxSerializedSize(members, current_alignment, false, false);
```

Reply:
Because it is a reference: `bool & is_key_unbounded`